### ISSUE TITLE
Infinity has a lot of unlabelled buttons, making the app inaccessible to screen reader users

### ISSUE THREAD

**Issue by LionessStrong** [Docile-Alligator/Infinity-For-Reddit#1398]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

version 5.4.1

### Where did you get the app from

Google Play

### Android version

13

### Device model

All models with screen readers turned on

### Steps to reproduce

1. Open settings, go to accessibility and turn on talkback.
2. 2. Open infinity and navigate to a random post.
3. 3. Try to find the upvoat/downvoat or any basic formatting options in the post creation screen with the help of the screen reader.

### Expected behaviour

Infinity has a lot of unlabelled buttons, making the navigational experience a hell for screen reader user like me.
, 
, Although unlabelled buttons are not a big issue for sighted members, screen reader users like me depend on it heavily. Whenever my screen reader encounters an unlabelled button, it just reads it as "button." How can I know what the button does without clicking it?
, 
, Buttons to create a post, format a post, upvoating and downvoating posts and many basic options are unlabelled. It will be great if the devs take some time in labelling all the buttons so screen reader users like me can have the same fluent experience of using a functional client.
, 
, Here is a helpful article to understand why unlabelled buttons ruin the experience of using an app. , https://support.google.com/accessibility/android/answer/7158690?hl=en
, 
, I really hope the devs fix this app. I will be keeping an eye for future updates.

### Current behaviour

Many options are not readable to screen readers as they are unlabelled. Whenever the screen reader encounters an unlabelled button, it just reads it simply as "button." With this info, a screen reader user will not understand what the button does if he presses it.

**Comment by atypicalfigure** [comment]
Hi, can I contribute on this?
